Tasks/Common/VstsAzureHelpers_/VstsAzureHelpers_.psm1

```diff
@@ -51,5 +51,41 @@ function Initialize-Azure {
     }
 }
 
+function Get-AzureCmdletsVersion
```

Comment:
This logic is inaccurate. This function isn't used anywhere. Please delete the function, or if you do need it then please let me know what behavior you're after and we can figure out an appropriate function.